CHANGES

```diff
@@ -22,7 +22,14 @@ Incompatible changes
 Deprecated
 ----------
 
+* LaTeX: some internals of the ``sphinxadmonition`` environment have been
+  marked for removal at 7.0.0 in the :file:`sphinxlatexadmonitions.sty`.
+  Custom re-definitions of ``sphinxlightbox`` or ``sphinxheavybox`` will have
+  to be updated if they depended upon them.
```

Comment:
@jfbu I accidentally didn't resolve this merge conflict properly - was this text intended to be removed as part of a7949d42903efa64bc5f8f870d39d811e272ecda?

Reply:
No problem, thanks for checking :+1:

Reply:
Thanks for conscienciousness ;-) :+1: